docroot/modules/custom/civictheme_migrate/MAPPINGS.md

```diff
@@ -2,39 +2,129 @@ This document is used as a starting point to define a "contract" between field
-| `banner`                         | Container for banner items              | `container` | No       | approved |                                              | 1.4                |
-| `banner.theme`                   | Banner theme                            |             | No       | approved | `field_c_n_banner_theme`                     | 1.4                |
-| `banner.title`                   | Banner title                            | `text`      | No       | approved | `field_c_n_banner_title`                     | 1.4                |
-| `banner.type`                    | Banner type                             | `text`      | No       | approved | `field_c_n_banner_type`                      | 1.4                |
-| `banner.blend_mode`              | Banner blend mode                       |             | No       | approved | `field_c_n_blend_mode`                       | 1.4                |
-| `banner.featured_image`          | Banner featured image                   | `Media`     | No       | approved | `field_c_n_banner_featured_image`            | 1.4                |
-| `banner.background`              | Banner background image                 | `Media`     | No       | approved | `field_c_n_banner_background_image`          | 1.4                |
-| `banner.hide_breadcrumb`         | Hide breadcrumb                         | `boolean`   | No       | approved | `field_c_n_banner_hide_breadcrumb`           | 1.4                |
-| `content`                        | Container for components                | `container` | No       | draft    | `field_c_n_components`                       | 1.4                |
-| `content.text_content`           | Container for Basic text component      | `container` | No       | draft    | `field_c_n_components`                       |                    |
-| `content.manual_list`            | Container for Manual list component     | `container` | No       | draft    |                                              |                    |
-| `content.accordion`              | Container for Accordion component       | `container` | No       | draft    |                                              |                    |
-| `content.attachment`             | Container for Attachment component      | `container` | No       | draft    |                                              |                    |
-| `content.attachment.title`       | Attachment component title              | `container` | No       | draft    | `field_c_n_components.field_c_p_title`       |                    |
-| `content.attachment.content`     | Attachment component content            | `container` | No       | draft    | `field_c_n_components.field_c_p_content`     |                    |
-| `content.attachment.attachments` | Attachment component attachments        | `container` | No       | draft    | `field_c_n_components.field_c_p_attachments` |                    |
-|                                  |
+## Legend
```

Comment:
go to this link to see how it looks 

https://github.com/salsadigitalauorg/civictheme_source/blob/eb4f1e5e6078d30ec8e596d8614e77e50c8fc342/docroot/modules/custom/civictheme_migrate/MAPPINGS.md